### DISCUSSION TITLE
CLN in MF6

### DISCUSSION THREAD

**Discussion by sebmeng** [MODFLOW-ORG/modflow6#1905] [Q&A]
Hello Community!
I just wanted to ask if there is any intention of including CLN in further versions of MF6. And maybe someone can share their experience in how to tackle the same kind of problems that requires CLN's (like complex well geometries, for example)  using the available packages of MF6.

Thanks in advance for any insight in this matter!

Cheers,
Sebastian

**Comment by p-ortega** [comment]
I think MAW could be an option? 

We have used it for inclined wells by getting the intersected cells and adding them to the connectiondata.

**Reply by ghost** [reply]
Hi Pablo!
This sounds like a nice alternative. Could you share a little bit more of that experience of tilted wells with MAW?

Greetings!
